docs/api/io/flow_control.md

```diff
@@ -2,51 +2,66 @@
 
-Here is an example that uses a ticker object:
+If you don't need the precision of a high-frequency ticker or timeout, we recommend that you use LowPowerTicker or LowPowerTimeout instead. This allows the system to be put in deep sleep mode.
 
-[![View code](https://www.mbed.com/embed/?url=https://github.com/ARMmbed/mbed-os-snippet-Flow-Control-Ticker)](https://github.com/ARMmbed/mbed-os-snippet-Flow-Control-Ticker/blob/v6.0/main.cpp)
+- _High-resolution microsecond ticker/timeout_ ([Ticker](ticker.html), [Timeout](timeout.html))
+- _Low Power ticker/timeout_ ([LowePowerTicker](lowpowerticker.html), [LowePowerTimeout](lowpowertimeout.html))
 
-<span class="tips">**Tip:** You may want to read the [power optimization](../apis/platform-concepts.html) tutorial to understand how to achieve power savings. </span>
+Usage of the low power classes will inform the operating system of your desire to enable Deep Sleep mode in your board, although enabling this depends on the specific environment and characteristics where your code runs. But it's helpful to know that there are options to enable this. For more information about Sleep and Deep Sleep, we would suggest to look at our documentation page [here](power-management-sleep.html) as well as watch our [Mbed Office Hours video](https://www.youtube.com/watch?v=OFfOlBaegdg&t=12s) where the concepts of Sleep and Deep Sleep are described in-depth.
 
 ### Thread
 
 If your application is running in RTOS mode then threads are another efficient way to blink an LED. During the waiting period, it is possible to take advantage of Mbed OS optimizations to automatically conserve power and deal with other tasks.
 
 [![View code](https://www.mbed.com/embed/?url=https://github.com/ARMmbed/mbed-os-snippet-Flow-Control-Thread)](https://github.com/ARMmbed/mbed-os-snippet-Flow-Control-Thread/blob/v6.0/main.cpp)
 
+Threads are the most efficient way to run tasks in Mbed OS since the operating system internally contains optimizations for efficiently scheduling the tasks and putting the device as much as possible to sleep (or deep sleep).
+
+### EventQueue
+
+The EventQueue class has a simple-to-use `call_every()` function to schedule repeated actions:
+
+[![View code](https://www.mbed.com/embed/?url=https://github.com/ARMmbed/mbed-os-examples-docs_only/blob/master/Tutorials_UsingAPIs/Flow-Control-EventQueue)](https://github.com/ARMmbed/mbed-os-examples-docs_only/blob/master/Tutorials_UsingAPIs/Flow-Control-EventQueue/main.cpp)
+
+For one-off delays, `call_in()` is available. Just as with Ticker/Timeout, if no threads are running during a wait, the system enters sleep mode.
+
+Alternatively, you can use a Ticker/Timeout to create delays (like the earlier example) but delegate the action (e.g. blinking) from an ISR to the EventQueue using `EventQueue::call()` to lift restrictions like `printf`. This strategy is explained in [The EventQueue API](../the-eventqueue-api.html) tutorial.
```

Comment:
I don't see any advantage from this use case over the pure equeue use case. And this has a rather heavy cognitive complexity as it involves more concepts/tools.

Reply:
Maybe the only exception is when we use EventQueue with a low power timer. But still, using an EventQueue to exit ISRs quickly is a general strategy which we already [document elsewhere](https://os.mbed.com/docs/mbed-os/v5.15/tutorials/the-eventqueue-api.html), so we may not need to explain anything here.

Reply:
I removed this paragraph, will push a commit later

Reply:
Done